react/utils/date/index.ts

```diff
@@ -0,0 +1,87 @@
+import moment from 'moment'
+
+export const getFirstTime = (date: Date) => {
```

Comment:
This could be simpler with `moment`: `export const getFirstTime = (date: Date) => moment(date).startOf('day').toDate()`